src/input.ts

```diff
@@ -53,6 +55,9 @@ export class Input {
 
     this.squashTitle = core.getInput('squash-title') === 'true'
```

Comment:
This should now set the `squashCommitTitle` and `this.squashCommitMessage ` templates, if `true`.